postcssRtlOptions.mjs

```diff
@@ -0,0 +1,64 @@
+export default {
+  prefixSelectorTransformer: (prefix, selector) => {
+    // To increase specificity https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity 
+    // for certain classnames within .sendbird-conversation__messages context.
+    // This only applies when the root dir is set as 'rtl' but forceLeftToRightMessageLayout is true.
+    if ([
+      // Normal message list
+      'sendbird-message-content',
+      // Thread message list
+      'sendbird-thread-list-item-content',
+      'sendbird-parent-message-info'
```

Comment:
How about making this Array as const? like this
```tsx
const CLASS_NAMES_TO_CHECK = [
  // Normal message list
  'sendbird-message-content',
  // Thread message list
  'sendbird-thread-list-item-content',
  'sendbird-parent-message-info'
];
```

```tsx
CLASS_NAMES_TO_CHECK.some(...)
```

Reply:
Yeah addresses in https://github.com/sendbird/sendbird-uikit-react/pull/1184/commits/52397fd40e6d71fb54851ed276da8c1d950cafa3